source/localizable/smart_cart/webhook.html.md.erb

```diff
@@ -113,6 +113,55 @@ Name           | Type   | Value | Description
 
 <%= partial 'localizable/smart_cart/order_object' %>
 
+## Test requests (sandbox)
+
+For [demo orders](/smart_cart/orders_api/#test-requests-sandbox), separate endpoints are available to trigger different
+kinds of webhook requests for testing. In the following examples, `:code` should be replaced with one of the following values:
+
+- DEMO-OPEN
+- DEMO-ACCEPTED
+- DEMO-REJECTED
+- DEMO-INVOICE
+- DEMO-INVOICE39A
+
+An order creation or order update event should be received at the webhook url of the shop, depending on the request action.
```

Comment:
I think we should add a bold/warning text here, explaining a couple extra points:

> Demo webhook requests are not instant, and may take from a few seconds up to a few minutes to be processed and sent. After getting a successful response (`200 OK`), you would just have to wait a bit, before trying another one. 
> All requests would be sent to the production Webhook URL that is already set-up in Merchants panel.

Feel free to adjust if it does not "read well" for you.